content/docs/projects/trust-manager.md

```diff
@@ -56,79 +70,299 @@ spec:
 
 #### Namespace Selector
 
-The target `namespaceSelector` can be used for scoping which Namespaces targets
-are synced to, supporting the field `matchLabels`. Please see
-[here](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors)
-for more information and how label selectors are configured.
+A target's `namespaceSelector` is used to restrict which Namespaces your `Bundle`'s target
+should be synced to.
 
-If `namespaceSelector` is empty, a bundle target will be synced to all
-Namespaces.
+`namespaceSelector` supports the field `matchLabels`.
 
----
+Please see [Kubernetes documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors)
+for more information about how label selectors can be configured.
+
+If `namespaceSelector` is empty, a `Bundle`'s target will be synced to all Namespaces.
+
+> ⚠️ A future update to trust-manager **will** change this behavior so that an empty namespace selector will sync only
+to the trust-manager namespace by default.
 
 ## Installation
 
-First, install [cert-manager](https://cert-manager.io/docs/installation/) to the
-cluster, and then the trust operator. It is advised to run the trust operator in
-the `cert-manager` namespace.
+### Demo
+
+If you've got Docker installed and you just want to play with trust-manager as soon as possible, we provide
+a `demo` command to get a [Kind cluster](https://kind.sigs.k8s.io/) set up with minimal fuss.
+
+```bash
+git clone --single-branch https://github.com/cert-manager/trust-manager trust-manager
+cd trust-manager
+make demo
+# kubeconfig is in ./bin/kubeconfig.yaml
+# kind cluster is called "trust"
+```
+
+The demo installation uses Helm, and roughly matches what you'd get by installing trust-manager into your own
+cluster using Helm - although it uses locally-built images rather than the ones we provide publicly.
+
+### Helm
+
+Helm is the easiest way to install trust-manager and comes with a publicly trusted certificate bundle package
+(for the`useDefaultCAs` source) derived from Debian containers.
+
+When installed via Helm, trust-manager has a dependency on cert-manager for provisioning an application certificate,
+and as such trust-manager is also installed into the cert-manager namespace.
 
 ```bash
 helm repo add jetstack https://charts.jetstack.io --force-update
 helm upgrade -i -n cert-manager cert-manager jetstack/cert-manager --set installCRDs=true --wait --create-namespace
 helm upgrade -i -n cert-manager trust-manager jetstack/trust-manager --wait
 ```
 
-#### Quick Start Example
+### Manual Installation
+
+We strongly recommend that you install trust-manager using Helm and we don't currently support manually installed
+versions of trust-manager. This is so that we can focus on continuing to improve trust-manager with the resources
+we currently have available.
+
+A longer term goal of ours is to add first-class support for installation without requiring Helm or cert-manager.
```

Comment:
can we remove this sentence, we probably don't want to add this in the docs. I think this will only lead to users expecting more and waiting for this to be completed. If we don't add this, they might be more inclined to contribute something like this themselves.